cognite/client/data_classes/workflows.py

```diff
@@ -1175,3 +1175,185 @@ def load(cls, resource: Any, cognite_client: CogniteClient | None = None) -> Wor
 
     def dump(self, camel_case: bool = True, as_external_id: bool = False) -> list[dict[str, Any]]:
         return [workflow_id.dump(camel_case, as_external_id_key=as_external_id) for workflow_id in self.data]
+
+
+WorkflowTriggerType: TypeAlias = Literal["schedule"]
+
+
+class WorkflowScheduledTriggerRule(CogniteObject):
+    """
+    This class represents a scheduled trigger rule.
+
+    Args:
+        cron_expression(str | None): The cron specification for the scheduled trigger.
+    """
+
+    def __init__(self, cron_expression: str | None = None) -> None:
+        self.trigger_type: WorkflowTriggerType = "schedule"
+        self.cron_expression = cron_expression
+
+
+class WorkflowTriggerCreate(CogniteResource):
```

Comment:
More a personal preference, but typically we have a `WorkflowTriggerCore`, which contains the shared parameter for the read and write format of the workflow trigger, that both `WorkflowTriggerCreate` and `WorkflowTrigger` subclasses.

Reply:
I applied the same pattern too.